templates/web/base/reports/_ward-list.html

```diff
@@ -1,16 +1,27 @@
 [% IF children.size %]
 <section id="council_wards" class="hidden-js">
 <h2>[% ward_text %]</h2>
-    <p>[% loc('Follow a ward link to view only reports within that ward.') %]</p>
+    <p class="js-ward-single">[% loc('Follow a ward link to view only reports within that ward.') %]</p>
+    <p class="js-ward-multi hidden">[% loc('Select multiple wards to view only reports within those wards.') %]</p>
+    <form method="get" action="">
     <section class="full-width">
     <ul class="item-list item-list--wards">
       [% FOR child IN children.values.sort('name') %]
         <li class="item-list__item item-list--wards__item">
-            <a href="[% child.url %]">[% child.name %]</a>
+          <label class="js-ward-multi hidden">
+            <input type="checkbox" name="ward" value="[% child.name | html %]">
```

Comment:
Alternatively to my previous comment, if you stored the short name (last bit of the URL) here you could just pass them through.